mobile/src/bundles/common/helpers/get-avatar-style/get-avatar-style.ts

```diff
@@ -0,0 +1,54 @@
+import {
+    IconSize,
+    PhotoType,
+    TextCategory,
+} from '~/bundles/common/enums/enums';
+import { type ValueOf } from '~/bundles/common/types/types';
+
+const DIVIDER = 2;
+
+type IconStyle = {
+    width: number;
+    height: number;
+    borderRadius: number;
+};
+
+type AvatarStyles = {
+    size: IconStyle;
+    font: ValueOf<typeof TextCategory>;
+    iconSize: number;
+};
```

Comment:
```suggestion
type IconStyle = {
    width: number;
    height: number;
    borderRadius: number;
};

type AvatarStyles = {
    size: IconStyle;
    font: ValueOf<typeof TextCategory>;
    iconSize: number;
};

const DIVIDER = 2;
```

Reply:
B28. All source files follow the recommended structure.
<img width="720" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/103518438/4c37c8de-ffff-4370-a9f5-c0d6d5f4e7ac">